src/components/qas/questions/qaFillableTable/FillableTableRangeInput.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+import { TextField } from 'react-md';
+import StellarValueRange from '../../../charts/shared/StellarValueRange';
+
+class FillableTableRangeInput extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      minValue: undefined,
+      maxValue: undefined,
+      hasFocus: false,
+      answerable: false,
+    };
+  }
+
+  componentDidMount = () => {
```

Comment:
use lifecycle hooks to get the answer out of the props if it has been answered and put it into the state, otherwise leave as undefined.